gen_subgraph_urls.py

```diff
@@ -0,0 +1,24 @@
+import json
+from bal_addresses.subgraph import Subgraph
+
+
+def main():
+    urls = {}
+
+    with open("extras/chains.json", "r") as f:
+        chains = json.load(f)
+    for chain in chains["CHAIN_IDS_BY_NAME"]:
+        if chain not in urls:
+            urls[chain] = {}
+        for subgraph_type in ["core", "gauges", "blocks", "aura"]:
+            subgraph = Subgraph(chain)
+            url = subgraph.get_subgraph_url(subgraph_type)
```

Comment:
Could we add some logic that tests that all the URLs are basically working, at least don't return a 404 when running some kind of a simple/nothing query?

If they don't maybe somehow signal that in the outputs?

Reply:
so this is one possible approach: 25b4cc3

Reply:
Yeah looks good.  As you can see the aura subgra[hs are incomplete, but that may be due to the lack of complete subgraph coverage by aura or another schema or something.  

In lieu of any docs/clear record of where to find consistent subgraph endpoints per chain, maybe we should just not include the ecosystem stuff in our library?   Will open another ticket to discuss this.